src/main/java/com/mongodb/jdbc/MongoSQLDatabaseMetaData.java

```diff
@@ -1892,29 +1885,34 @@ private BsonDocument getFunctionColumnValuesDoc(
             int i,
             String argName,
             String argType,
-            boolean isReturnColumn) {
+            boolean isReturnColumn)
+            throws SQLException {
+
+        BsonTypeInfo bsonTypeInfo =
+                argType == null ? BSON_NULL : BsonTypeInfo.getBsonTypeInfoByName(argType);
```

Comment:
Should we be using the new helper method from the parent class, `getFunctionParameterValues`? This seems to mostly be the same code, if not exactly the same code.